server/server/src/lib.rs

```diff
@@ -94,6 +94,14 @@ pub async fn start_server(
     let auth = auth_data(&settings.server, token_bucket, token_secret, &certificates);
     info!("Initialising server context..done");
 
+    // LOGGING
+    let service_context = service_provider.basic_context().unwrap();
+    let log_level = service_provider
+        .log_service
+        .log_level(&service_context)
+        .unwrap();
+    logging_init(settings.logging.clone(), None, log_level);
```

Comment:
Moved from main so that log level can be updated

Reply:
oh. darn - I'd like to have migration still logged, and can't see how to do that.
the logger can only be initialised once, and doesn't have the option to update during runtime. 
I wondered about adding another logger, that logs the init stage, which is removed when the `fast_log` is initialised. But a little clumsy, I think.

what about switching out the `fast_log`? We could use [simple_log](https://docs.rs/simple-log/1.6.0/simple_log/) which supports file rotation, file size.. and 🎉  runtime log level setting! and config!

We could configure with defaults that log the migrations and then update the config when we hit this point. Of course if the migration logging isn't in the same directory, then we can't read those logs via the API.. but at least the data is captured. If you think the idea has legs, then I'd recommend a new issue rather than making changes to this one.

Reply:
Will have a look and create a new issue for it (: